x-pack/platform/plugins/shared/triggers_actions_ui/public/application/sections/rules_list/components/notify_badge/notify_badge.tsx

```diff
@@ -323,7 +325,10 @@ export const RulesListNotifyBadge: React.FunctionComponent<RulesListNotifyBadgeP
     if (isSnoozed) {
       return snoozedButton;
     }
-    return unsnoozedButton;
+    if (isRuleEditable) {
+      return unsnoozedButton;
+    }
+    return null;
```

Comment:
I found this a bit confusing. I think it is better if at the top of the `useMemo` we do:

```
 if (!isScheduled && !isSnoozed && !isSnoozedIndefinitely && !isRuleEditable) {
      return null;
    }
```

and leave the rest of the logic as it is. Wdyt?